edb/pgsql/metaschema.py

```diff
@@ -94,6 +111,118 @@ def __init__(self) -> None:
         ),
 
 
+class AlterCurrentDatabaseSetString(dbops.Function):
+    """Alter a configuration parameter of the current database."""
```

Comment:
I'd clarify with

```suggestion
    """Alter a PostgreSQL configuration parameter of the current database."""
```

for this and the following functions.